velox/docs/functions/spark/string.rst

```diff
@@ -89,6 +89,16 @@ Unless specified otherwise, all functions return NULL if at least one of the arg
 
     Returns the length of ``string`` in characters.
 
+.. spark:function:: levenshtein(string, string[, threshold]) -> integer
```

Comment:
nit: use different names for the two input strings as they are not the same variable.